CHANGES.md

```diff
@@ -8,6 +8,9 @@ To be released.
 
 ### Deprecated APIs
 
+ -  Removed `IStore.GetCanonicalGenesisBlock<T>()` interface method and all its
+    implementations.  [[#2664]]
```

Comment:
Shouldn't it be placed under <q>Backward-incompatible API changes</q>?

Reply:
You mean because the change is `interface` related?

Reply:
Fixed!